test/fork/smoke-test/Smoke_Base.t.sol

```diff
@@ -0,0 +1,175 @@
+
+  ICalculator internal calculator;
+
+  IBotHandler internal botHandler;
+
+  IEcoPythCalldataBuilder internal ecoPythBuilder;
+
+  ProxyAdmin internal proxyAdmin;
+
+  address internal constant OWNER = 0x6409ba830719cd0fE27ccB3051DF1b399C90df4a;
+  address internal constant POS_MANAGER = 0xF1235511e36f2F4D578555218c41fe1B1B5dcc1E; // set market status;
+  address internal ALICE;
+  address internal BOB;
+
+  function setUp() public virtual {
+    ALICE = makeAddr("Alice");
+    BOB = makeAddr("BOB");
+
+    vm.createSelectFork(vm.envString("ARBITRUM_ONE_FORK"), 130344667);
+
+    // -- LOAD FORK -- //
+    vm.startPrank(OWNER);
+    ecoPyth = IEcoPyth(0x8dc6A40465128B20DC712C6B765a5171EF30bB7B);
+    tradeHelper = ITradeHelper(0x963Cbe4cFcDC58795869be74b80A328b022DE00C);
+    tradeService = ITradeService(0xcf533D0eEFB072D1BB68e201EAFc5368764daA0E);
+    botHandler = IBotHandler(0xD4CcbDEbE59E84546fd3c4B91fEA86753Aa3B671);
+    liquidationService = ILiquidationService(0x34E89DEd96340A177856fD822366AfC584438750);
+    liquidationReader = ILiquidationReader(0x9f13335e769208a2545047aCb0ea386Cce7F5f8F);
+
+    configStorage = IConfigStorage(0xF4F7123fFe42c4C90A4bCDD2317D397E0B7d7cc0);
+    perpStorage = IPerpStorage(0x97e94BdA44a2Df784Ab6535aaE2D62EFC6D2e303);
+    vaultStorage = IVaultStorage(0x56CC5A9c0788e674f17F7555dC8D3e2F1C0313C0);
+
+    proxyAdmin = ProxyAdmin(0x2E7983f9A1D08c57989eEA20adC9242321dA6589);
+    ecoPythBuilder = IEcoPythCalldataBuilder(0x4c3eC30d33c6CfC8B0806Bf049eA907FE4a0AB4F); // UnsafeEcoPythCalldataBuilder
+
+    // -- LOAD FORK -- //
+    vm.stopPrank();
+
+    // -- UPGRADE -- //
+    vm.startPrank(proxyAdmin.owner());
+    Calculator newCalculator = new Calculator();
+    proxyAdmin.upgrade(
+      TransparentUpgradeableProxy(payable(0x0FdE910552977041Dc8c7ef652b5a07B40B9e006)),
+      address(newCalculator)
+    );
+
+    // -- LOAD UPGRADE -- //
+    vm.stopPrank();
+
+    calculator = ICalculator(0x0FdE910552977041Dc8c7ef652b5a07B40B9e006);
+  }
+
+  function _getSubAccount(address primary, uint8 subAccountId) internal pure returns (address) {
+    return address(uint160(primary) ^ uint160(subAccountId));
+  }
+
+  function _getPositionId(address _account, uint8 _subAccountId, uint256 _marketIndex) internal pure returns (bytes32) {
+    address _subAccount = _getSubAccount(_account, _subAccountId);
+    return keccak256(abi.encodePacked(_subAccount, _marketIndex));
+  }
+
+  function _setTickPriceZero()
+    internal
+    view
+    returns (bytes32[] memory priceUpdateData, bytes32[] memory publishTimeUpdateData)
+  {
+    int24[] memory tickPrices = new int24[](34);
+    uint24[] memory publishTimeDiffs = new uint24[](34);
+    for (uint i = 0; i < 34; i++) {
+      tickPrices[i] = 0;
+      publishTimeDiffs[i] = 0;
+    }
+
+    priceUpdateData = ecoPyth.buildPriceUpdateData(tickPrices);
+    publishTimeUpdateData = ecoPyth.buildPublishTimeUpdateData(publishTimeDiffs);
+  }
+
+  function _setPriceData()
```

Comment:
This function set all prices to 1. The current name is not so informational on this. Consider changing function name or add some comments to explain. Or make this function receive price that need to be set.